samples/asymmetricDecrypt.js

```diff
@@ -46,8 +46,12 @@ async function asymmetricDecrypt(
     name: name,
     ciphertext: ciphertextBuffer,
   });
+  const plaintext = result.plaintext.toString('utf8');
```

Comment:
In the asymmetric samples in other languages, we assume that the data to be encrypted or signed over is binary data.  Is there a strong rationale for assuming string data in Node?  I'd suggest that our inputs and outputs should all be of type Buffer.

This also would hopefully circumvent all the questions around "when should I use base64?" which seems to be a common source of programmer confusion.  In the GAPIC clients, there really shouldn't be a need to use or think about base64 encoding.

Reply:
This is what the other samples do? I was kinda just following their example.

Reply:
@bdhess @sethvargo I personally like that our Node.js samples allow folks to invoke the samples from the command-line, and play with them. The JavaScript community is very tooling oriented, so this fits our brains 😆 

Perhaps we could switch the sample to take a file path, and use `createReadStream` to pass a stream to the API rather than decoding the bytes?

Reply:
The feedback I've heard from the (KMS) community is that they don't want to need to use files when possible.

Reply:
I too have a preference for not writing to files, but I don't consider myself a Node expert or super in-touch with the needs of Node developers, so I don't feel like my opinion on that is super relevant.

But AFAIK that is a separate issue from whether to convert to strings, or whether to base64-encode. See #242 for a pull request that cleans this up in the existing samples.

Reply:
Okay I see what you mean. @bcoe can we merge brads PR and then I'll update these samples to use files?

Reply:
:+1: done, flag me (probably after the holidays) as soon as this is ready.

Reply:
@bcoe flag 😄